src/functions/instance/getTableData.js

```diff
@@ -76,6 +92,9 @@ export default async ({ schema, table, filtered, pageSize, onlyCached, sorted, p
     }
   }
 
+  dynamicAttributesFromDataTable = getAttributesFromTableData(newData, allAttributes)
+  allAttributes.push(...dynamicAttributesFromDataTable);
+
   // sort columns, but keep primary key / hash attribute first, and created and updated last.
   // NOTE: __created__ and __updated__ might not exist in the schema, only include if they exist.
   const orderedColumns = allAttributes.filter((a) => ![hashAttribute, '__createdtime__', '__updatedtime__'].includes(a)).sort();
```

Comment:
In testing this, one thing I realized is that while we are intentionally sorting the columns so that extra columns appear on the right and least frequently used columns are on the furthest right, this is all undone by the sort call on this line. I kind of think we should remove this sort(), as I think there is more value in ordering columns by frequency/declaration order rather than just alphabetic.

Reply:
Done!